src/GafferUI/BackdropNodeGadget.cpp

```diff
@@ -113,8 +113,17 @@ std::string BackdropNodeGadget::getToolTip( const IECore::LineSegment3f &line )
 	}
 
 	const Backdrop *backdrop = static_cast<const Backdrop *>( node() );
-	std::string title = backdrop->titlePlug()->getValue();
-	std::string description = backdrop->descriptionPlug()->getValue();
+	std::string title = "Error evaluating title";
```

Comment:
Seems like a big enough chunk of logic that we should put it in a function rather than repeat it? We don't have a good mechanism for getting a context into the NodeGraph right now, but at some point we'd probably want to do that, at which point having things consolidated in one spot would help.